openvdb_houdini/houdini/GEO_VDBTranslator.cc

```diff
@@ -70,12 +70,16 @@
 #include <UT/UT_IStream.h>
 #include <UT/UT_Version.h>
 
+#include <FS/FS_IStreamDevice.h>
```

Comment:
Hi @jmlait - this header isn't available in Houdini 15.0 or earlier, would it be possible to add some #ifdefs around the new functionality please so we can still build VDB against earlier versions? (@OpenVDB-DevTeam)